extensions/attention-stream/metrics.yaml

```diff
@@ -182,3 +182,140 @@ user_journey:
+    expires: never
+
+  survey_id:
+    type: uuid
+    description: |
+      A survey ID for this user
+    lifetime: ping
+    send_in_pings:
+      - article-contents
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1778196
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1778367
+    notification_emails:
+      - than@mozilla.com
+    expires: never
+
+  url:
+    type: url
+    description: |
+      URL for the current page
+    lifetime: ping
+    send_in_pings:
+      - article-contents
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1778196
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1778367
+    notification_emails:
+      - than@mozilla.com
+    expires: never
+
+  title:
+    type: string
+    description: |
+      Title of the current page
+    lifetime: ping
+    send_in_pings:
+      - article-contents
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1778196
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1778367
+    notification_emails:
+      - than@mozilla.com
+    expires: never
+
+  text_content:
+    type: text
+    description: |
+      Contents of current page
+    lifetime: ping
+    send_in_pings:
+      - article-contents
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1778196
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1778367
+    notification_emails:
+      - than@mozilla.com
+    expires: never
+
+advertisements:
+  page_id:
+    type: uuid
+    description: |
+      A unique ID for this visit
+    lifetime: ping
+    send_in_pings:
+      - advertisements
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1778196
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1778367
+    notification_emails:
+      - than@mozilla.com
+    expires: never
+
+  url:
+    type: url
+    description: |
+      URL for the current page
+    lifetime: ping
+    send_in_pings:
+      - advertisements
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1778196
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1778367
+    notification_emails:
+      - than@mozilla.com
+    expires: never
+
+  body:
+    type: text
+    description: |
+      Body of the current page
+    lifetime: ping
+    send_in_pings:
+      - advertisements
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1778196
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1778367
+    notification_emails:
+      - than@mozilla.com
+    expires: never
+
+  ads:
+    type: text
+    description: |
+      Ads
+    lifetime: ping
+    send_in_pings:
+      - advertisements
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1778196
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1778367
+    notification_emails:
+      - than@mozilla.com
+    expires: never
```

Comment:
I based this on what the code is actually doing, I think they must not have actually implemented everything in that schema:

https://github.com/mozilla-rally/beyond_the_paywall/blob/6b62e691aea0ddaff1457c0c582be0122b84dfd3/src/Advertisements.js#L51-L58

The actual schema that was presumably loaded for this collection specifies things like `tabId` but that's not actually send AFAICT, and I don't think we need it because we have UserJourney and it has `pageId`:

https://github.com/mozilla-services/mozilla-pipeline-schemas/blob/main/schemas/pioneer-sgsb-beyond-the-paywall/advertisement/advertisement.1.schema.json

Let's follow up on this, I think we're probably OK for now but we might want to expand upon this. I used `text` for these to give us more flexibility in exactly what metadata we collect.